### PR TITLE
Set latest node 10.19 as default and only, removing 6.9.2

### PR DESCRIPTION
**Pull request by VictorVelarde**
#15530

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -198,8 +198,8 @@
     "moment-v3": "global:moment"
   },
   "engine": {
-    "node": "^10.15.1",
```
Comment (dgaubert) [inline package.json]: Our cloud instances are provisioned with Node.js `v10.15.1`.  If you bump the version in the `engine` section the process will produce warnings when this package is installed as a dependency.

Comment (VictorVelarde) [reply package.json]: Okay, I knew I had to ping you for CR 😄 

One question: is that 10.15.1 used for a particular reason? Can we somehow coordinate with @CartoDB/infrastructure-team  so we all use latest LTS (10.19.0)?

For development we (FE) are currently using 10.19, to build assets. Also, in docker env, I have recently update the docker, so LTS is used.

I'm not trying to mess around, but would sticking to the same version be useful?

Comment (jvillarf) [reply package.json]: Hi @VictorVelarde , i'm not the one to decide about nodeJS versions, @dgaubert is the **man** for that. There is a procedure for **updating** to new node versions (http://doc-internal.cartodb.net/platform/common_platform_operations/upgrade_nodejs_cloud.html?highlight=blue%20green), that @dgaubert already used in the past.

Comment (dgaubert) [reply package.json]: We should use the same version for every service in our stack for the shake of simplicity. If not upgrades can be a pain, I wish to upgrade to Node 12 indeed.  I'm trying to convince managers for that but I'm afraid it's not important enough from their point of view.

So sorry but we are stuck in version `10.15.1` for the moment.

Comment (VictorVelarde) [reply package.json]: I'm ok with it. Thx a lot! I didn't want to create a problem, but fix potential ones.

**Comment by VictorVelarde** [comment]
I had forgotten about this small one... 

So then I'm gonna just remove node 6.9.2 and leave 10.15.1 as default for building assets, and that's it.

**Pull request merged**